<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>640</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>ObiLink</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralWidget{background-image: url(:/images/images/bg.png)}</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="DropListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>266</y>
      <width>320</width>
      <height>283</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border-style:outset;
color:white;</string>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_qrcode">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>17</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_remove">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>577</y>
      <width>94</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_remove{
background-image: url(:/images/images/btn_remove_n.png);
border-style: outset;
border-color: beige;}
#btn_remove:pressed{background-image: url(:/images/images/btn_remove_p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_startshare">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>29</y>
      <width>94</width>
      <height>41</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_startshare{
background-image: url(:/images/images/btn_start_n.png);
border-style: outset;
border-color: beige;}
#btn_startshare:pressed{background-image: url(:/images/images/btn_start_p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stopshare">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>29</y>
      <width>94</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_stopshare{
background-image: url(:/images/images/btn_stop_p.png);
border-style: outset;
border-color: beige;}
#btn_stopshare:pressed{background-image: url(:/images/images/btn_stop_n.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>86</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>공유주소</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_dropbox">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>215</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>577</y>
      <width>94</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_add{
background-image: url(:/images/images/btn_add_n.png);
border-style: outset;
border-color: beige;}
#btn_add:pressed{background-image: url(:/images/images/btn_add_p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tb_addr">
    <property name="geometry">
     <rect>
      <x>158</x>
      <y>102</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border-style:outset;
color:white;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;IP Address:8888&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DropListWidget</class>
   <extends>QListWidget</extends>
   <header>droplistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
